src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+you use the placeholder to show the correct format for values such as dates. This can be especially 
+problematic for users on phones, particularly if they're distracted or feeling stressed! 
+{% endAside %}
+
+#### Make buttons helpful {: #html-button}
+
+Use [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)
+for buttons! You can also use `<input type="submit">`, but there's no point in using a div or some 
+other random element acting as a button. Button elements provide accessible behaviour and built-in 
+form submission functionality, and can easily be styled. 
+
+Give each form submit button a value that says what it does. For each step towards checkout, use 
+a descriptive call-to-action that shows progress and makes the next step obvious. For example, label 
+the submit button on your delivery address form **Proceed to Payment** rather than **Continue** 
+or **Save**.
+
+{: #disable-submit}
+
+Consider disabling a submit button once the user has tapped or clicked it—especially when the user is 
+making payment or placing an order. Many users click buttons repeatedly, even if they're working fine. 
+That can mess up checkout and add to server load. 
+
+On the other hand, don't disable a submit button waiting on complete and valid user input. For 
+example, don't just leave a **Save Address** button disabled because something is missing or invalid. 
+That doesn't help the user—they may continue to tap or click the button and assume that it's broken. 
+Instead, explain to the user what's gone wrong and what they need to do if they attempt to submit 
+the form with invalid data. This is particularly important on mobile, where data entry is more 
+difficult and missing or invalid form data may not be visible on the user's screen by the time they 
+attempt to submit a form.
+
+### Make the most of HTML attributes {: #html-attributes}
+
+#### Make it easy for users to enter data
+
+{: #type-attribute}
+
+Use the appropriate input [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) 
+to provide the right keyboard on mobile and enable basic built-in validation by the browser. 
+
+For example, use `type="email"` for email addresses and `type="tel"` for phone numbers.
+
+<figure class="w-figure">
+  <img src="images/input-type.jpg" alt="Two screenshots of Android phones, showing a keyboard appropriate 
+  for entering an email addresss (using type=email) and for entering a telephone number (with type=tel).">
+  <figcaption class="w-figcaption">Keyboards appropriate for email and telephone.</figcaption>
+</figure>
+
+{: #inputmode-attribute}
+
+{% Aside 'warning' %}
+Using type="number" adds an up/down arrow to increment numbers, which makes no sense for data such as 
+telephone, payment card or account numbers.
+
+Instead, you should set `type="text"` (or leave off the attribute, since `text` is the default) and 
+specify `inputmode="numeric"`, to get a numeric keyboard on mobile. 
```

Comment:
```suggestion
Instead, to get a numeric keyboard on mobile set `type="text"` (or leave off the attribute, since `text` is the default) and specify `inputmode="numeric"`. 
```